File: project104.c

#include <stdio.h>
#include <math.h>

int main()
{
  int sum = 0;
  int sum1 = 0;
  int sum2 = 0;
  int count = 541;
  sum1 = (pow(1+sqrt(5),count) / pow(2,count));
  sum1 = sum1 * ((5 + sqrt(5)) / 10);
  sum2 = (pow((1-sqrt(5)),count) / pow(2,count));
  sum2 = sum2 * ((5-sqrt(5))/10);
  sum = sum1 + sum2 + 1;
  printf("%d\n",sum);
}

File: project104.jl

sum = big(0)
sum1 = big(1)
sum2 = big(1)
99987
n = 3

function ispen(num)
    array = [1,2,3,4,5,6,7,8,9]
    last = num%1000000000
    d = sort(digits(last))
    len = floor(log10(num))+1
    if len <= 9 return false end
    l = d[len-8:len]
    d = d[1:9]
    if array == d && array == sort(l) return true end
    return false
end

while(true)
    global n
    global sum
    global sum1
    global sum2
    sum = sum1+sum2
    if(n==32468)
        println("finish = $n")
        println("$sum")
        break
    end
    println("$n")
    sum2 = sum1
    sum1 = sum
    n += 1
end


